e2e/rnTestUtilities.js

```diff
@@ -1,12 +1,26 @@
 require('coffeescript').register();
 
 const _ = require('lodash');
+const nodeUtil = require('util');
 
 db = require('../../heap/back/db');
 testUtil = require('../../heap/test/util');
 
 const HEAP_ENV_ID = '2084764307';
 
+const waitIfIos = async () => {
+  if (device.getPlatform() === 'ios') {
```

Comment:
Be nice to just get rid of this!  Have you looked into the server arg we need to start the collector with?

Reply:
I have, but I haven't been able to get it to work as expected :(